src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import React from 'react';
+import { TabType } from '../../types/Tab';
+import { Tab } from '../Tab';
+
+type Props = {
+  tabs: TabType[];
+  selectedTabId: string;
+  onTabSelected: (tab: TabType) => void;
+};
+
+export const Tabs: React.FC<Props> = (props) => {
+  const {
+    tabs,
+    selectedTabId,
+    onTabSelected,
+  } = props;
+
+  const selectedTab = tabs.find(tab => selectedTabId === tab.id) || tabs[0];
+  const { content } = selectedTab;
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            return (
+              <Tab
+                key={tab.id}
+                tab={tab}
+                onTabSelected={onTabSelected}
+                selectedTabId={selectedTab.id}
+              />
+            );
+          })}
```

Comment:
```suggestion
          {tabs.map(tab =>  (
              <Tab
                key={tab.id}
                tab={tab}
                onTabSelected={onTabSelected}
                selectedTabId={selectedTab.id}
              />
            );
          )}
```
avoid useless return